settings.py

```diff
@@ -12,3 +12,8 @@
 # To filter by a cookie or `table_id` range [(value1, value2)]
 CONSISTENCY_COOKIE_IGNORED_RANGE = []
 CONSISTENCY_TABLE_ID_IGNORED_RANGE = []
+
+# Retries options for `kytos/core.openflow.connection.error`
+CONN_ERR_MAX_RETRIES = 3
+CONN_ERR_MIN_WAIT = 1  # minimum wait between iterations in seconds
```

Comment:
We might more feedback fro production usage here, but this way, assuming that in a worst case a control plane is lkely 250ms away that might leave some room to reasonably work for this case too, and considering some network connection flaps that might last a second or so, and the last accumulated retry up to 4 seconds isn't too long to the point of start to interfere with consistency checks. Let me know if you'd have any suggestion or other values you'd rather have.

Reply:
Also, another feedback in general here regarding the overall settings.py configuration that each NApp uses, maybe in the future it might be worth allowing setting these values with a higher preference via environment variables, if any specific env var is set, that way, if in production an operator needs to adapt these NApps settings then it's just a matter of redeploying kytos overwriting the environment variable instead of having to change the source code.

Reply:
Also, another feedback in general here regarding the overall settings.py configuration that each NApp uses, maybe in the future it might be worth allowing setting these values with a higher preference via environment variables, if any specific env var is set, that way, if in production an operator needs to adapt these NApps settings then it's just a matter of redeploying kytos overwriting the environment variable instead of having to change the source code. Let me know what you think and if we should support this.

Reply:
@viniarck, having those parameters configurable through the settings is great and I think the values you chose were very reasonable. On top of them, the network operator can customizer as needed. For instance, if one want to have the MIN_WAIT starting on a few hundred milliseconds, they could just set `CONN_ERR_MIN_WAIT = 0.200`

Reply:
> Also, another feedback in general here regarding the overall settings.py configuration that each NApp uses, maybe in the future it might be worth allowing setting these values with a higher preference via environment variables, if any specific env var is set, that way, if in production an operator needs to adapt these NApps settings then it's just a matter of redeploying kytos overwriting the environment variable instead of having to change the source code. Let me know what you think and if we should support this.

Yes, great idea! Indeed we need enhance the settings approach currently being used. Maybe we could map another issue on kytos core to rethink the way settings are being used.

Reply:
Sounds good, appreciate your review @italovalcy and @ajoaoff. I've captured this issue on kytos core just so we can evolve it and keep refining https://github.com/kytos-ng/kytos/issues/153